libs/core/src/lib/radio/radio-button/radio-button.component.ts

```diff
@@ -34,6 +34,37 @@ export class RadioButtonComponent implements OnChanges, AfterViewInit, CssClassB
     @ViewChild('inputElement')
     inputElement: ElementRef;
 
+    /**
+     * Attached to the aria-label attribute of the host element. In most cases, aria-labelledby will
+     * take precedence so this may be omitted.
+     */
+    @Input('aria-label')
+    ariaLabel = '';
+
+    /**
+     * Users can specify the `aria-labelledby` attribute which will be forwarded to the input element
+     */
+    @Input('aria-labelledby')
+    ariaLabelledby = null;
+
+    /** The 'aria-describedby' attribute is read after the element's label and field type. */
+    @Input('aria-describedby')
+    ariaDescribedby: string;
+
+    /** The 'aria-disabled' for giving accessibility for disabled radio element. */
+    @Input('aria-disabled')
+    ariaDisabled: boolean;
+
+    /** sets radio tooltip */
+    @Input()
+    title: string;
+
+    /**
+     * Includes the radio in the page tab sequence.
+     */
+    @Input()
+    tabIndex: number;
```

Comment:
@Attribute

Reply:
tabindex for Radio button changes.
for selected radio it should be 0 and for not selected radios, it should be -1.
it will change dynamically once selection of choice changes.
So have to keep it as @Input().
same is done in materials design implementation as well.